test/db/cmd/types

```diff
@@ -384,10 +384,10 @@ t const_reference
 tc const_reference
 EOF
 EXPECT=<<EOF
-const_reference = const_reference
-typedef const_reference const_reference;
-pf 
-const_reference;
+pf d a
+struct const_reference {
+	int a;
+};
 EOF
```

Comment:
Because there has been a change in the save strategy for types, the later type with the same name in the DWARF will overwrite the earlier one.

In this particular test, the `struct const_reference` from later overwrites the `typedef const_reference` from earlier.

Reply:
You can change this test to do the self-reference test from the commands, without loading any file.

Reply:
Don't forget this one